bom/runtime/pom.xml

```diff
@@ -34,6 +34,13 @@
                 <scope>import</scope>
             </dependency>
 
+            <!-- Debezium Outbox Extension -->
+            <dependency>
+                <groupId>io.debezium</groupId>
+                <artifactId>debezium-quarkus-outbox-deployment</artifactId>
```

Comment:
Do you have a runtime bom?

Reply:
@aloubyansky Not that I am aware, we simply have this hierarchy of poms:

* `debezium-parent`
  * `debezium-quarkus-outbox-parent` 
     * `debezium-quarkus-outbox-deployment`
     * `debezium-quarkus-outbox`

Is this necessary?

Reply:
What's the purpose of `debezium-quarkus-outbox` in this case? How is it different from the deployment one? In Quarkus and the platform we follow a naming convention `name` and `name-deployment` are runtime and deployment artifacts/boms.
Here, it appears that you are importing your deployment bom into the runtime bom of the platform.

Reply:
Hi @aloubyansky, I believe commit ba166c2 should address this.  I also took the liberty to bump the dependency to our latest CR1 release.

Reply:
Ok. About CR1, we are not yet releasing anything but Final releases of the platform. So whatever you include may end up being exposed in the final release of the platform. You should include the version you are comfortable with. You should also rebase your PR.

Reply:
@aloubyansky During rebase I noticed a new `other` directory added for integration tests where the qpid-jms artifact is now located.  Should I move the `debezium-outbox` artifact under the `other` directory or leave it where it is presently?